frontend/src/pages/Workflows/WorkflowContainer.js

```diff
@@ -157,7 +157,9 @@ const mapDispatchToProps = (dispatch, ownProps) => {
     openHistory: () => {
       dispatch(openHistory());
     },
-    openWorkflowDetails: (pId, sId, id) => dispatch(fetchWorkflowitem(pId, sId, id)),
+    openWorkflowDetails: (pId, sId, id, worflowDetailsInitialTab) => {
```

Comment:
Consider converting to arrow function:

```suggestion
 openWorkflowDetails: (pId, sId, id, worflowDetailsInitialTab) => dispatch(fetchWorkflowitem(pId, sId, id, true, worflowDetailsInitialTab))
    
```